packages/agent-toolkit/src/core/tools/platform-api-tools/workforms-tools/workforms.types.ts

```diff
@@ -354,15 +360,13 @@ export interface Form {
   active: boolean;
   title: string;
   ownerId?: number;
-  createWithAI: boolean;
   builtWithAI: boolean;
```

Comment:
Just curious - what do you do with this information? 
Thinking if we can learn from your usage here